release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3257,17 +3257,10 @@ In the following tables, features are marked with the following statuses:
 
 * {product-title} version {product-version} disconnected installation fails on {ibm-power-name} Virtual Server. (link:https://issues.redhat.com/browse/OCPBUGS-36250[*OCPBUGS-36250*])
 
-* In {hcp} for {product-title}, if you disable the xref:../installing/overview/cluster-capabilities.adoc#ingress-operator_cluster-capabilities[Ingress capability], the Console Operator returns the following error message:
-+
-[source,text]
-----
-RouteHealthAvailable: failed to GET route.
-----
-+
-To avoid this error, do not disable the `Ingress` capability in an {product-title} managed cluster. (link:https://issues.redhat.com/browse/OCPBUGS-33788[*OCPBUGS-33788*])
-
 * If you have IPsec enabled on the cluster, on the node hosting the north-south IPsec connection, restarting the `ipsec.service` systemd unit or restarting the `ovn-ipsec-host` pod causes a loss of the IPsec connection. (link:https://issues.redhat.com/browse/RHEL-26878[*RHEL-26878*])
 
+* If you set the `baselineCapabilitySet` field to `None`, you must explicitly enable the Ingress Capability, because the installation of a cluster fails if the Ingress Capability is disabled. (link:https://issues.redhat.com//browse/OCPBUGS-33794[*OCPBUGS-33794*])
```

Comment:
```suggestion
* If you set the `baselineCapabilitySet` field to `None`, you must explicitly enable the Ingress Capability, because the installation of a cluster fails if the Ingress Capability is disabled. (link:https://issues.redhat.com/browse/OCPBUGS-33794[*OCPBUGS-33794*])
```

consolidating a doubled `//`.